introspection-engine/connectors/mongodb-introspection-connector/tests/test_api/mod.rs

```diff
@@ -0,0 +1,89 @@
+use datamodel::common::preview_features::PreviewFeature;
+use introspection_connector::{IntrospectionConnector, IntrospectionContext, Warning};
+use mongodb::{Client, Database};
+use mongodb_introspection_connector::MongoDbIntrospectionConnector;
+use names::Generator;
+use once_cell::sync::Lazy;
+use std::future::Future;
+use tokio::runtime::Runtime;
+
+pub use bson::doc;
+pub use expect_test::expect;
+
+pub static CONN_STR: Lazy<String> = Lazy::new(|| {
+    std::env::var("TEST_DATABASE_URL").expect("Please set TEST_DATABASE_URL env var pointing to the MongoDB instance.")
+});
```

Comment:
Super nitpicky, but does this error nicely? I think we had to do that to abort on the first failing tests in the `test-setup` based setup: https://github.com/prisma/prisma-engines/blob/e52940a463af3be120869f463a7b64797807422b/libs/test-setup/src/test_api_args.rs#L99

Reply:
It does not. Fixing.